src/client/components/AddPeopleToRoom/AddPeopleToRoomForm.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './AddPeopleToRoom.css';
+import PropTypes from 'prop-types';
+
+export default function AddPeopleToRoom({
+  DiplayListOFUsers,
+  displayUserInGroup,
+  users,
+  addUsers,
+  onChangeInput,
+  nextToGroup,
+}) {
+  return (
+    <div className="container">
+      <div className="header">
+        <div className="list-name">
+          <div>click on + to add people in room</div>
+          <div>
+            <span>
+              {addUsers.length}/{users.length}
+            </span>
+          </div>
+        </div>
+      </div>
+
+      <div className="inputsearch">
+        <input type="text" placeholder="Search" onChange={onChangeInput} />
+        <button type="button" className="next-button" onClick={nextToGroup}>
+          Next
+        </button>
+      </div>
+
+      <div className="add-people">{displayUserInGroup()}</div>
```

Comment:
Why is this lower case and the other uppercase? 

All functions are lower case. This looks a little weird. Why are you calling a function here? Why not just have a component?
`<DisplayUserInGroup />`